common/remote/sockfile.cpp

```diff
@@ -7028,32 +7090,41 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
         {
             Owned<ISocket> sock;
             Owned<ISocket> sockSSL;
+            Owned<ISocket> acceptedRSSock;
             bool sockavail = false;
             bool securesockavail = false;
-            if (_connectMethod == SSLNone)
+            bool rowServiceSockAvail = false;
+            if (_connectMethod == SSLNone && (nullptr == rowServiceSock.get()))
                 sockavail = acceptsock->wait_read(1000*60*1)!=0;
```

Comment:
outside the scope of this change, but it would be nice to #define this read wait value and perhaps make it configurable